src/models/UserRequestActions.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/**
+ * @author Nicolas CARPi <nico-git@deltablot.email>
+ * @copyright 2024 Nicolas CARPi
+ * @see https://www.elabftw.net Official website
+ * @license AGPL-3.0
+ * @package elabftw
+ */
+
+declare(strict_types=1);
+
+namespace Elabftw\Models;
+
+use Elabftw\Elabftw\Db;
+use Elabftw\Enums\Action;
+use Elabftw\Enums\RequestableAction;
+use Elabftw\Enums\State;
+use Elabftw\Exceptions\ImproperActionException;
+use Elabftw\Interfaces\RestInterface;
+use PDO;
+
+/**
+ * Request action for users
+ */
+class UserRequestActions implements RestInterface
```

Comment:
why are the exceptions not translatable?

Reply:
Let's keep them like this for now. Eventually, they'll go away if we split RestInterface into little chunks, so we don't have to add such methods, it's code smell. I also want to limit the number of translated strings, especially for users hitting undocumented api endpoint, it's fine to talk to them in english.